csrc/ir/utils.cpp

```diff
@@ -1310,4 +1310,111 @@ MmaOpDetails getMmaOpDetails(
   return details;
 }
 
+// Verifies the assumptions made when evaluating a fusion containing MmaOp:
+// 1. MmaOp is preceded by a broadcast.
+// 2. The inputs to MmaOp are broadcasted as the last dim for the first operand
+// and the first dim for the second operand. The inputs of MmaOp will be [M, K,
+// 1] x [1, K, N].
+void verifyMmaOpForEvaluation(MmaOp* mma_op, DataType final_out_dtype) {
```

Comment:
```suggestion
void verifyMmaOpForEvaluation(MmaOp* mma_op, const DataType expected_in_dtype) {
```

I understand where `final_out_dtype` comes from, but I think `expected_in_dtype` is a better name for the context of this function.